lib/led-matrix/include/LEDMatrix.h

```diff
@@ -0,0 +1,37 @@
+/* LED Matrix Header */
+#pragma once
+#include <mbed.h>
+#include <CANMsg.h>
+class LEDMatrix
+{
+  public:
+	// Constructor: An LED Matrix is defined by the pins it is connected to
+	LEDMatrix(PinName R, PinName G, PinName B);
+	
+	// This function takes in values from 0 to 255 for each color stream and maps them to a PWM signal
+	void setColor(uint8_t R, uint8_t G, uint8_t B);
```

Comment:
I'd rather have different function names for some of these functions that explain a bit about their functionality eg. setColor and setColorRGB. It will make the code easier to read there'll be less confusion as to what parameters are needed to call the function.